spec/services/builders/chartkick/project_distribution_data_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
```

Comment:
I think you can delete this require as you have the `.rspec` file 😉

Reply:
i am not sure why but if a remove the requirement it stops recognizing the RSpec.describe declaration 🤔

Reply:
Nice catch, maybe we can edit the `.rspec` file to be:

```
--require rails_helper
```

Reply:
i tested two cases, when i run `bundle exec rspec ./spec/services/builders/chartkick/project_data_spec.rb` it fails.. but running `bundle exec rspec ./spec` pass, maybe it could be a configuration stuff?

Reply:
since all the specs in the project are requiring the rails_helper i think it would be better something doing a task to remove them all, what do you think?

Reply:
🤔 may be... anyway, this is not a requirement for this PR, we can address it later 👍